modules/persistent-storage-csi-cinder-storage-class.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-cinder.adoc
+
+[id="persistent-storage-csi-cinder_{context}"]
+= Making OpenStack Cinder CSI the default storage class
+
+In {product-title}, the default storage class references the in-tree Cinder driver. The storage class will default to referencing OpenStack Cinder CSI in a subsequent update of {product-title}. Volumes provisioned using the existing in-tree storage class are planned for migration to the OpenStack Cinder CSI storage class at that time.
+
+The OpenStack Cinder CSI driver uses the `cinder.csi.openstack.org` parameter key to support dynamic provisioning.
+
+.Procedure
+To enable the use of OpenStack Cinder CSI in {product-title} {product-version}, you must overwrite the default in-tree storage class by applying the `standard-csi` storage class.
+
+. List the StorageClass:
++
+[source,terminal]
+--
+$ oc get storageclass
+--
++
+[source,terminal]
+.Example output
+--
+NAME                   PROVISIONER                RECLAIMPOLICY   VOLUMEBINDINGMODE      ALLOWVOLUMEEXPANSION   AGE
+standard(default)      cinder.csi.openstack.org   Delete          WaitForFirstConsumer   true                   46h
+standard-csi           kubernetes.io/cinder       Delete          WaitForFirstConsumer   true                   46h
+--
+
+. Change the value of the annotation `storageclass.kubernetes.io/is-default-class` to `false` for the default storage class, as shown in the following example:
++
+[source,terminal]
+--
+$ oc patch storageclass standard -p '{"metadata": {"annotations": {"storageclass.kubernetes.io/is-default-class": "false"}}}'
+--
+
+. Make another storage class the default by adding or modifying the annotation as `storageclass.kubernetes.io/is-default-class=true`.
++
+[source,terminal]
+--
+$ oc patch storageclass standard-csi -p '{"metadata": {"annotations": {"storageclass.kubernetes.io/is-default-class": "true"}}}'
+--
+
+. Verify that the PVC is now referencing the CSI storage class by default:
++
+[source,terminal]
+--
+$ oc get storageclass
+--
++
+[source,terminal]
+.Example output
+--
+NAME                   PROVISIONER                RECLAIMPOLICY   VOLUMEBINDINGMODE      ALLOWVOLUMEEXPANSION   AGE
+standard               kubernetes.io/cinder       Delete          WaitForFirstConsumer   true                   46h
+standard-csi(default)  cinder.csi.openstack.org   Delete          WaitForFirstConsumer   true                   46h
+--
+
+. Optional: To create a PVC that is provisioned by OpenStack Cinder CSI, you can now create a `PersistentVolumeClaim` object without specifying the storage class:
++
+[source,yaml]
+--
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: cinder-claim
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
+  storageClassName: <1>
```

Comment:
I don't suggest to have this line, not specifying a specific storage class is different from storageClassName:"".  
You can keep the note somehow.

Reply:
Made some small tweaks here, and added the command to create the PVC.